src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -209,12 +206,7 @@ export const WalletsPage = () => {
               <SidebarMenuButton title={t('tokens')} to={`/wallets/${selectedAccount?.address}/tokens`} />
               <SidebarMenuButton title={t('nfts')} to={`/wallets/${selectedAccount?.address}/nfts`} />
               <SidebarMenuButton title={t('transactions')} to={`/wallets/${selectedAccount?.address}/transactions`} />
-              <SidebarMenuButton
-                title={t('connections')}
-                type="button"
```

Comment:
You won't need this prop anymore, so Rename the component to <SidebarMenuLink /> and remove this prop and the condicional from the component to always render a link